### ISSUE TITLE
Not available via Integrations.

### ISSUE THREAD

**Issue by gjws** [bremor/bureau_of_meteorology#180]
I'm fairly new to HA, but following the standard process to add an integration and this doesn't appear when searching for "Bureau of Meteorology". Has this integration been removed?

**Comment by grouts112** [comment]
@gjws  try following the instructions here
https://github.com/bremor/bureau_of_meteorology

**Comment by lordsnipe1** [comment]
I'm also experiencing a similar issue, I have manually added the folder into the custom_components folder, and my home assistant log states: "WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration bureau_of_meteorology which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant".

So I believe I have installed it correctly (as per all my other custom components), however when I try and use the "add integration" window to configure, it doesn't show up. I've tried to refresh the page a number of times, no change.

Any ideas?

EDIT: I updated Home Assistant and now it's showing.
